src/clr-angular/forms/styles/_checkbox.clarity.scss

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
+
+.clr-control-container {
```

Comment:
Cant we extract the basic styles outside of the `.clr-control-container` selector? Positioning, margin styles can go inside of the container but other styles can be moved out of it right?

Reply:
I did this intentionally to ensure that people used the correct DOM structure. Let's chat about this more.